### PR TITLE
print("Hello again, Zainab!!")

### PR DESCRIPTION
**Pull request by wanda-carlson**
print("Hello again, Zainab!!")

### CODE REVIEW

**Comment by wanda-carlson** [comment]
coolio

**Comment by wanda-carlson** [comment]
slackcoolio

File: src/bert.py [hunk 1]
```diff
@@ -18,3 +18,4 @@
 print("Hello Test")
 print("Hello Test")
```
Comment (henry-richard) [inline src/bert.py:19]: add something here

File: src/bert.py [hunk 2]
```diff
@@ -18,3 +18,4 @@
...
 print("Hello again, Zainab!!")
```
Comment (henry-richard) [inline src/bert.py:20]: and here..

**Review by henry-richard — CHANGES_REQUESTED** [review]

**Review by henry-richard — APPROVED** [review]
lgtm

**Pull request merged**